Main.java

```diff
@@ -9,7 +10,29 @@
    * @param args the command line arguments
    */
   public static void main(String[] args) {
-    // TODO code application logic here
+    // create a Scanner for user input 
+    Scanner input = new Scanner (System.in);
+    // ask for the x coordinate of the first point 
+    System.out.println ("Please enter in the x coordinate of the first point");
+    //declare a variable to store x1
+    int x1 = input.nextInt();
+    //ask for the y coordinate of the first point 
+    System.out.println ("Please enter in the y coordinate of the first point");
+    //declare a variable to store y1
+    int y1 = input.nextInt();
+    //ask for the x coordinate of the second point 
+    System.out.println ( "Please enter in the x coordinate of the second point");
+    //declare a variable to store x2
+    int x2 = input.nextInt();
+    //ask for the y coordinate of the second point
+    System.out.println ("Please enter in the y coordinate of the second point");
+    //declare a variable to store y2
+    int y2 = input.nextInt();
+    // declare a variable to store slope 
+    double slope = (y2 - y1) / (x2 - x1);
```

Comment:
IS it possible for a point to have an x or y value that contains a decimal place? These should probably be doubles instead of ints. We know this because the sample output shows the slope as 2.0 instead of just 2. Otherwise, this is great looking code!